### ISSUE TITLE
Add mechanism to copy configurations

### ISSUE THREAD

**Issue by harningt** [jp-bennett/Fwknop2#45]
I have a few applications / ports that need setup to a single machine and want to setup multiple hosts, however the current implementation requires I either setup a QR code to partly fill details in, or manually fill it all in multiple times.

A long-press => duplicate option would save quite a bit of time.

**Comment by jp-bennett** [comment]
I agree that the handling of configs needs a bit more polish, and this in particular is a good idea.

In the current version, this can be accomplished by editing a saved config, and just saving it under a different name.  This leaves the old config intact, and creates a new config under the new name.

I'll try to refine this for the next version.
